tests/kernel/update_kernel.pm

```diff
@@ -106,7 +111,14 @@ sub update_kernel {
     }
     else {
         # Use single patch or patch list
-        zypper_call("in -l -t patch $patches", exitcode => [0, 102, 103], log => 'zypper.log', timeout => 1400);
+        if (is_transactional) {
+            # Proceed with transactional-update patch
+            trup_call("patch");
+            # Reboot system after patch, to make sure that further checks are done on updated system
+            reboot_on_changes;
+        } else {
+            zypper_call("in -l -t patch $patches", exitcode => [0, 102, 103], log => 'zypper.log', timeout => 1400);
+        }
```

Comment:
I guess instances of this is where the follow ups to https://progress.opensuse.org/issues/117028 are to be added, right?

something to note would be that `patch` is also something that has to be there too.

@fgerling Can you add a ticket to our backlog about this?

Reply:
> something to note would be that `patch` is also something that has to be there too.

I made comment in  mentioned ticket regarding `patch`.